stable/newrelic-infrastructure/templates/daemonset.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels: {{ include "newrelic-infra.labels" . | indent 4 }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+  name: {{ template "newrelic-infra.fullname" . }}
+spec:
+  template:
+    metadata:
+    {{- if .Values.podAnnotations }}
+      annotations:
+{{ toYaml .Values.podAnnotations | indent 8}}
+    {{- end }}
+      labels: {{ include "newrelic-infra.labels" . | indent 8 }}
+    spec:
+      hostPID: true
+      hostIPC: true
+      hostNetwork: true
+      containers:
+        - name: {{ template "name" . }}
+          image: "{{.Values.image.name}}:{{ .Values.image.tag }}"
+          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+          securityContext:
+            privileged: true
+          command: [ "bash", "-c", "source /etc/kube-nr-infra/config && /usr/bin/newrelic-infra -config /etc/newrelic/newrelic-infra.yml" ]
+          {{- if or (.Values.verboseLog) (.Values.logFile) }}
+          env:
+          {{- if .Values.verboseLog }}
+          - name: NRIA_VERBOSE
+            value: 1
+          {{- end }}
+          {{- if .Values.logFile }}
+          - name: NRIA_LOG_FILE
+            value: {{ .Values.logFile }}
+          {{- end }}
+          {{- end }}
+          volumeMounts:
+            - name: config
+              mountPath: /etc/newrelic/
+            - name: newrelic-config
+              mountPath: /etc/kube-nr-infra
+              readOnly: true
+            - name: dev
+              mountPath: /dev
+            - name: run
+              mountPath: /var/run/docker.sock
+            - name: log
+              mountPath: /var/log
+            - name: host-root
+              mountPath: /host
+              readOnly: true
+          {{- if .Values.resources }}
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
+          {{- end }}
+      volumes:
+        - name: newrelic-config
+          secret:
+            secretName: newrelic-config
+        - name: dev
+          hostPath:
```

Comment:
Consistently indent with two spaces.

Reply:
👍

Reply:
Make list indentation consistent as well, i. e. content below `env` should be indented one more level.

Reply:
👍

Reply:
Also fixed now.